app/scripts/modules/core/src/search/infrastructure/searchResultPods.component.ts

```diff
@@ -0,0 +1,6 @@
+import { module } from 'angular';
```

Comment:
I'm not fond of adding one file per `react2angular` component.

Reply:
It's good for you.